src/components/cc-orga-member-card/cc-orga-member-card.js

```diff
@@ -103,8 +91,19 @@ export class CcOrgaMemberCard extends withResizeObserver(LitElement) {
     /** @type {Ref<CcSelect>} */
     this._roleRef = createRef();
 
-    /** @type {string} Set by `withResizeObserver` mixin. See the `onResize` method for more info. */
-    this._size = '';
+    /** @type {number|null} Set from the willUpdate method triggered by the `ResizeController`. */
+    this._width = null;
+
+    /**
+     * @type {ResizeController}
+     * used to:
+     * - wrap buttons when the component width is below 740.
+     * - switch to a vertical card layout when the width is below 580.
+     * - show buttons and badges below each other when the width is below 350.
+     */
+    new ResizeController(this, {
+      width: [BREAKPOINT_TINY, BREAKPOINT_SMALL, BREAKPOINT_MEDIUM],
+    });
```

Comment:
issue: let's keep a reference to the controller so we can get its `width` property

```js
this._resizeController = new ResizeController(this);
```